lib/tasks/group.rake

```diff
@@ -0,0 +1,17 @@
+namespace :group do
```

Comment:
Should we add this to a migration?  Seems like if we don't run it on initial deployment, we'll lose a bunch of access until we do.